core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -840,7 +840,7 @@ public static boolean isValidTablePropertyKey(String key) {
   public static final EnumSet<Property> HOT_PATH_PROPERTIES = EnumSet.of(Property.TSERV_CLIENT_TIMEOUT, Property.TSERV_TOTAL_MUTATION_QUEUE_MAX,
       Property.TSERV_ARCHIVE_WALOGS, Property.GC_TRASH_IGNORE, Property.TSERV_MAJC_DELAY, Property.TABLE_MINC_LOGS_MAX, Property.TSERV_MAJC_MAXCONCURRENT,
       Property.REPLICATION_WORKER_THREADS, Property.TABLE_DURABILITY, Property.INSTANCE_ZK_TIMEOUT, Property.TABLE_CLASSPATH,
-      Property.MASTER_METADATA_SUSPENDABLE);
+      Property.MASTER_METADATA_SUSPENDABLE, Property.TABLE_FAILURES_IGNORE, Property.TABLE_SCAN_MAXMEM);
```

Comment:
I really think this HOT_PATH_PROPERTIES thing is confusing. We should probably build this static set from properties which have a specific annotation... an annotation with a good javadoc and a better name than "Hot Path".